debian/control

```diff
@@ -27,15 +27,13 @@ Build-Depends:
  libfribidi-dev,
  libgl1-mesa-dev,
  libgbm-dev,
- libgme-dev,
```

Comment:
https://github.com/mpv-player/mpv-build/blob/f667554e23d754c0619603a1b88fe8d49a56318a/debian/rules#L44

but I agree it's kind of useless. just like `libgsm`